intro/models.md

```diff
@@ -151,21 +151,24 @@ As you can see in the example, due to the different column types, the values can
 | `geolocation`                                  | `object`                                                                               | `{"lng": 8.23..., "lat": 50.00...}` or<br/>`{"country_region": "Sweden"}`                                                                                             | Depending on column setting.                                                                                                                            |
 | `formula`                                      | `string`                                                                               | `"Albert Summer"`                                                                                                                                                     | Contains only the result of the formula.                                                                                                                |
+| `auto-number`                                  | `string`                                                                               | `"0001"`                                                                                                                                                              |                                                                                                                                                         |
 | `button`                                       | `null`                                                                                 | `null`                                                                                                                                                                | Always `null`                                                                                                                                           |
-| `link`                                         | `array`                                                                                | `[{"row_id": "Mr-3QnQ5Q-eeEqFBevtOkw", "display_value": "Google News"}]`                                                                                              |                                                                                                                                                         |
+| `link`                                         | `array`                                                                                | `["Mr-3QnQ5Q-eeEqFBevtOkw"]`                                                                                                                                          | Array of row IDs.                                                                                                                                       |
```

Comment:
I just double-checked this and the format depends on the endpoint (`listRowsDeprecated` vs `listRows`).

The new `listRows` endpoint actually returns  an array of objects.